interface/batchcom/emailnotification.php

```diff
@@ -73,45 +96,57 @@
-        <br>
-        <?php xl('Email Subject','e')?> :
-        <INPUT TYPE="text" NAME="email_subject" size="60" value="<?php echo $email_subject?>">
-        <br>
-
-        <?php xl('Name of Provider','e')?> :
-        <INPUT TYPE="text" NAME="provider_name" size="40" value="<?php echo $provider_name?>">
-        <br>
-        <?php xl('SMS Text, Usable Tag: ***NAME***, ***PROVIDER***, ***DATE***, ***STARTTIME***, ***ENDTIME***<br> i.e. Dear ***NAME***','e')?>
-        <br>
-        <TEXTAREA NAME="message" ROWS="8" COLS="35"><?php echo $message?></TEXTAREA>
-        <br><br>
-        <INPUT TYPE="submit" name="form_action" value="Save">
-    </div>
-</div>
-</FORM>
+    <?php include_once("batch_navigation.php");?>
+    <header class="text-center">
+        <h1>
+            <?php xl('Batch Communication Tool','e'); ?>
+            <small><?php xl('Email Notification','e')?></small>
+        </h1>
+    </header>
+    <main class="container">
+        <?php if ($form_err) { echo "<div class=\"alert alert-danger\">".xl("The following errors occurred").": $form_err</div>"; } ?>
+        <?php if ($sql_msg) { echo "<div class=\"alert alert-info\">".xl("The following errors occurred").": $sql_msg</div>"; } ?>
+        <form name="select_form" method="post" action="">
+            <input type="Hidden" name="type" value="Email">
+            <input type="Hidden" name="notification_id" value="<?php echo $notification_id;?>">
+            <div class="row">
+                <div class="col-md-12">
+                    <label for="email_sender"><?php xl('Email Sender','e')?>:</label>
+                    <input type="text" name="email_sender" size="40" value="<?php echo $email_sender; ?>" placeholder="sender name">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    <label for="email_subject"><?php xl('Email Subject','e')?>:</label>
+                    <input type="text" name="email_subject" size="40" value="<?php echo $email_subject; ?>" placeholder="email subject">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    <label for="provider_name"><?php xl('Name of Provider','e')?>:</label>
+                    <input type="text" name="provider_name" size="40" value="<?php echo $provider_name; ?>" placeholder="provider name">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    <label for="message"><?php xl('SMS Text, Usable Tags: ','e'); ?>***NAME***, ***PROVIDER***, ***DATE***, ***STARTTIME***, ***ENDTIME*** (i.e. Dear ***NAME***):</label>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-12">
+                    <textarea cols="35" rows="8" name="message"><?php echo $message; ?></textarea>
```

Comment:
Not sure why original developer chose 8x35 layout.  That makes sense for SMS but not email.  We have typically use a large html message here.  Ideally this should take up maximum available space without looking ugly.
  
_Wishlist alert_ : @bradymiller has recently brought in CKEditor.  It would be cool if users can format a rich text message inline.